src/main/java/roomescape/controller/AdminController.java

```diff
@@ -0,0 +1,19 @@
+package roomescape.controller;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Controller
+@RequestMapping("/admin")
+public class AdminController {
+    @GetMapping("")
```

Comment:
추가 path가 없을 땐 아예 파라미터를 생략해도 됩니다~!
```suggestion
    @GetMapping
```